packages/components/src/Combobox/hooks/useComboboxAccessibility.ts

```diff
@@ -0,0 +1,112 @@
+import { useEffect, useRef } from "react";
+import { useRefocusOnActivator } from "@jobber/hooks/useRefocusOnActivator";
+import { useFocusTrap } from "@jobber/hooks/useFocusTrap";
+import { usePopper } from "react-popper";
```

Comment:
We should reconsider our usage of `react-popper` moving forward. It's not a library I'm familiar with and when I looked into it, I found it's based on @popperjs/core, but that team has recently deprecated that library in favour of the new floating-ui, which has their own react-bindings if we chose to move over. `react-popper` itself hasn't seen an update in over a year, so I think, it's entered a soft deprecated state of its own where the maintainers realize there's no future for it.

That said, if you dig into the core of `floating-ui` it's only a handful of functions that I understand pretty well, so if we wanted, we could roll our own solution that mimics the react-popper API so we don't really have to migrate, just roll our own solution and start using that instead.

Reply:
good to know! yeah we've been using `react-popper` for a while now, I'm sure when it was introduced it was much more actively maintained. sounds like a bit of tech debt for us to replace all those instances with the newer library and hope it doesn't have the same problem in a year or two